llvm-node.d.ts

```diff
@@ -173,6 +173,16 @@ declare namespace llvm {
     isAllOnesValue(): boolean;
   }
 
+  class ConstantExpr extends Constant {
```

Comment:
It still seems like your ConstantExpr changes are in this PR, Maybe its best if you create a new branch only containing the getTypeAllocSize changes?